juniper_official/Chassis/check-ksyncd-core.rule

```diff
@@ -0,0 +1,54 @@
+/*
+ * Check presence of Ksyncd core files
+ *
+ * Checks if Ksyncd core files are present in /var/tmp on the other RE.
+ * 
+ * While replicating a FIB state, if Ksyncd (Kernel state synchronization daemon)
+ * gets an error from the backup routing engine kernel, it generates a ksyncd core
+ * and Kernel live vmcores for debugging.
+ *
+ * This rule detects the presence of Ksyncd core(s) and reports a yellow alarm.
+ */
+iceberg {
+    topic chassis.gres {
+        rule check-ksyncd-core {
+            synopsis "Check presence of Ksyncd core files";
+            description "Check if Ksyncd core files are present in /var/tmp on the other RE";
+            sensor ksyncd-core-sensor {
+                iAgent {
+                    file find_ksyncd_cores.yml;
+                    table checkKsyncdCoresTable;
+                    frequency 15m;
+                }
+            }
+            field ksyncd-cores {
+                sensor ksyncd-core-sensor {
+                    path ksyncd_cores;
+                }
+                description "This field captures number of Ksyncd cores on the device";
+            }
+            trigger report-ksyncd-cores {
```

Comment:
1) Provide trigger description and synopsis detailing sensor info.
2) Trigger frequency is must as default frequency is 10s and trigger gets evaluated once in 10s. but here sensor frequecy is 10 mins hence trigger frequency can be greaterthan equal to 10 mins.
e.g. synopsis "Routing engine GRES kpi";
       description "sets health based on Routing engine CPU utilization for 15 minutes";

Reply:
Addressed